aci/convert/convert.go

```diff
@@ -326,27 +347,74 @@ func (s serviceConfigAciHelper) getAciFileVolumeMounts(volumesCache map[string]b
 	return aciServiceVolumes, nil
 }
 
-func (s serviceConfigAciHelper) getAciSecretsVolumeMount() *containerinstance.VolumeMount {
-	if len(s.Secrets) == 0 {
-		return nil
+func (s serviceConfigAciHelper) getAciSecretsVolumeMounts() ([]containerinstance.VolumeMount, error) {
```

Comment:
I didn't try to refactor this in detail, but would it make sense to refactor getAciSecretsVolumeMounts() and getAciSecretVolumes() , and have a single method returning the []Volume and []VolumeMount ?

Reply:
that might make more sense for unit testing as well, since both methods are impacted by various scenarios